src/transform-sdk/js/schema_registry_example/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "schema_registry_example",
+  "type": "module",
+  "private": true,
+  "scripts": {
+    "build": "node esbuild.js"
+  },
+  "dependencies": {
+    "@redpanda-data/transform-sdk": "file:../module",
+    "@redpanda-data/transform-sdk-sr": "file:../sr-module",
+    "avsc": "github:mtth/avsc#7cb76a6eb7b0d5b5a71c9cfb446acf3eff763006"
```

Comment:
`npm` doesn't seem to accept a branch name on its own - has to be either a commit hash or a tag I guess?

Reply:
if you drop it then it defaults to the main branch. I am fine pinning - either way

Reply:
what npm version are you using? whenever I try to install this package from a branch (no tag or commit hash), i trip over [this](https://github.com/npm/cli/issues/3149), which is supposedly fixed, but I haven't had the patience to really look hard at it.

```
$ npm --version
10.8.1
```

Reply:
This is just from memory. Don't worry about it - this is fine 👍